doc/api/tls.md

```diff
@@ -898,6 +898,17 @@ It may be useful for debugging.
 
 See [Session Resumption][] for more information.
 
+### tlsSocket.isSessionReused()
+<!-- YAML
+added: v0.5.6
+-->
+
+* {boolean}
```

Comment:
I think the idea behind @vsemozhetbyt’s comment was to remove this line?

Reply:
Hm, I just pressed all the "commit suggestions" buttons and assumed that did what he suggested. I'll read through and see if they worked. I'm not loving this github feature.